src/App/components/Resource/components/Examples/index.js

```diff
@@ -0,0 +1,190 @@
+import React, {Component} from 'react'
+import {map, includes, size, uniqueId} from 'lodash'
+import {
+  smallContainerWidth,
+  mediumContainerWidth,
+  largeContainerWidth,
+  xlargeContainerWidth,
+  xxlargeContainerWidth,
+} from 'package/utils/hardCodedSizes'
+import Heading from 'package/components/Heading'
+import Button from 'package/components/Button'
+
+const containerWidthActions = [
+  {
+    label: 'Small',
+    containerWidth: smallContainerWidth,
+  },
+  {
+    label: 'Medium',
+    containerWidth: mediumContainerWidth,
+  },
+  {
+    label: 'Large',
+    containerWidth: largeContainerWidth,
+  },
+  {
+    label: 'XLarge',
+    containerWidth: xlargeContainerWidth,
+  },
+  {
+    label: 'XXLarge',
+    containerWidth: xxlargeContainerWidth,
+  },
+]
+
+const containerBackgroundActions = [
+  {
+    label: 'Dark',
+    containerBackground: 'dark',
+  },
+  {
+    label: 'Light',
+    containerBackground: 'light',
+  },
+]
+
+const containerBoxSizing = 8
+
+export const optOuts = [
+  'containerWidth',
+  'containerBackground',
+  'types',
```

Comment:
Types refers to any resource input: "arguments" for functions or "props" for components